release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -166,6 +166,11 @@ The following metrics are sent through telemetry.
 
 |===
 
+[id="ocp-4-12-nw-aws-load-balancer-operator"]
+==== AWS Load Balancer Operator
+
+In {product-title} 4.12, the AWS Load Balancer controller now implements the Kubernetes ingress specification for multiple matches. If multiple paths within an Ingress match a request, the longest matching path gets the precedence. If two paths still match, paths with an exact path type over a prefix path type gets the precedence.
```

Comment:
* Should "ingress" be capitalized in the first sentence?
* The phrase "gets the precedence" sounds odd to me. I think saying "takes precedence" is a more common phrase, so that's what I'd recommend (two places in this line).

Reply:
Addressed